### DISCUSSION TITLE
What is the End of LIfe for Prisma 3?

### DISCUSSION THREAD

**Discussion by mkarric-slingshot** [prisma/prisma#18807] [Q&A] [answered]
I am considering upgrading to version 4 and this would be another vote (among many) to support making that decision.

Thank you!

**Comment by nurul3101** [comment, marked as answer]
Hello @mkarric-slingshot 👋 

Once a new Prisma version is released, the older version doesn't receive any patch updates. So once version 3.1 is released, version 3.0 won't receive any updates.

Upgrading to Prisma 4 would indeed be a wise decision, as it will provide you with the latest enhancements and optimizations that come with it. Here's a [reference](https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-4) for upgrading to Prisma 4.

**Reply by mkarric-slingshot** [reply]
Thank you so much! ❤️

**Reply by nurul3101** [reply]
No worries! :)
